contracts/SideChain/Mint.es

```diff
@@ -0,0 +1,24 @@
+{
+  // SideChain claiming sERG contract
+
+  val mainchainUtxoState = getVar[AvlTree](125).get
+  val mainchainUtxoProof = getVar[Coll[Byte]](126).get
```

Comment:
124 , as 126 is used for P2SH on the main-chain, and likely sidechain would do the same